cmd/kots/cli/set-config.go

```diff
@@ -132,14 +133,19 @@ func SetConfigCmd() *cobra.Command {
 			}
 
 			response := struct {
-				Error string `json:"error"`
+				Error            string                                   `json:"error"`
+				ValidationErrors []configtypes.ConfigGroupValidationError `json:"validationErrors,omitempty"`
 			}{}
 			_ = json.Unmarshal(respBody, &response)
 
 			if resp.StatusCode != http.StatusOK {
 				if resp.StatusCode == http.StatusNotFound {
 					return errors.Errorf("app with slug %s not found", appSlug)
 				} else {
+					if len(response.ValidationErrors) > 0 {
+						logConfigValidationErrors(response.ValidationErrors, log)
+						return (errors.New(response.Error))
```

Comment:
Is it always the case that there will be a response error when there are validation errors?
```suggestion
						return errors.New(response.Error)
```